html/pfappserver/root/static.alt/src/globals/configuration/pfConfigurationWrixLocations.js

```diff
@@ -0,0 +1,1019 @@
+  'httpd.portal.access',
+  'httpd.portal.catalyst',
+  'httpd.proxy.error',
+  'httpd.proxy.access',
+  'httpd.webservices.error',
+  'httpd.webservices.access',
+  'httpd.api-frontend.access',
+  'api-frontend.log',
+  'pfstats.log',
+  'packetfence.log',
+  'pfbandwidthd.log',
+  'pfconfig.log',
+  'pfdetect.log',
+  'pfdhcplistener.log',
+  'pfdns.log',
+  'pffilter.log',
+  'pfmon.log',
+  'pfsso.log',
+  'radius-acct.log',
+  'radius-cli.log',
+  'radius-eduroam.log',
+  'radius-load_balancer.log',
+  'radius.log',
+  'redis_cache.log',
+  'redis_ntlm_cache.log',
+  'redis_queue.log',
+  'redis_server.log',
+  'mariadb_error.log',
+  'haproxy_portal.log',
+  'haproxy_db.log',
+  'etcd.log'
+]
+
+export const pfConfigurationWrixLocationsListColumns = [
+  { ...pfConfigurationListColumns.id, ...{ label: i18n.t('WRIX Identifer') } }, // re-label
+  pfConfigurationListColumns.buttons
+]
+
+export const pfConfigurationWrixLocationsListFields = [
+  { ...pfConfigurationListFields.id, ...{ text: i18n.t('WRIX Identifer') } } // re-text
+]
+
+export const pfConfigurationWrixLocationsListConfig = (context = {}) => {
+  return {
+    columns: pfConfigurationWrixLocationsListColumns,
+    fields: pfConfigurationWrixLocationsListFields,
+    rowClickRoute (item, index) {
+      return { name: 'wrixLocation', params: { id: item.id } }
+    },
+    searchPlaceholder: i18n.t('Search by Wrix Identifier'),
+    searchableOptions: {
+      searchApiEndpoint: 'wrix_locations',
+      defaultSortKeys: ['id'],
+      defaultSearchCondition: {
+        op: 'and',
+        values: [{
+          op: 'or',
+          values: [
+            { field: 'id', op: 'contains', value: null }
+          ]
+        }]
+      },
+      defaultRoute: { name: 'wrixLocations' }
+    },
+    searchableQuickCondition: (quickCondition) => {
+      return {
+        op: 'and',
+        values: [{
+          op: 'or',
+          values: [
+            { field: 'id', op: 'contains', value: quickCondition }
+          ]
+        }]
+      }
+    }
+  }
+}
+
+export const pfConfigurationWrixLocationViewFields = (context) => {
+  const {
+    isNew = false,
+    isClone = false
+  } = context
+  return [
+    {
+      tab: i18n.t('Identification'),
+      fields: [
+        {
+          label: i18n.t('Id'),
+          fields: [
+            {
+              key: 'id',
+              component: pfFormInput,
+              attrs: {
+                disabled: (!isNew && !isClone)
+              },
+              validators: {
+                [i18n.t('Value required.')]: required,
+                [i18n.t('Maximum 255 characters.')]: maxLength(255),
+                [i18n.t('Wrix Location exists.')]: not(and(required, conditional(isNew || isClone), hasWrixLocations, wrixLocationExists))
+              }
+            }
+          ]
+        },
+        {
+          label: i18n.t('Provider Identifier'),
+          fields: [
+            {
+              key: 'Provider_Identifier',
```

Comment:
Why mixing snake case and title case? Please adjust all keys to snake case only.

Reply:
Title case is used right through to the db schema (https://github.com/inverse-inc/packetfence/blob/devel/db/pf-schema-8.3.0.sql#L1001). Should we modify it there first (another PR)?

Reply:
Does it matter? I though column names would be case insensitive.